rllab/algos/batch_polopt.py

```diff
@@ -107,7 +108,9 @@ def __init__(
     def start_worker(self):
         self.sampler.start_worker()
         if self.plot:
-            plotter.init_plot(self.env, self.policy)
+            plotter.init_plot(self.env, self.policy, self.max_path_length)
+            # Call rollout once to display glfw window
+            rollout(self.env, self.policy, animated=True, max_path_length=self.max_path_length)
```

Comment:
Make a TODO/github issue to make this API better. Shouldn't this be part of init_plot()?